redis-om-spring/src/main/java/com/redis/om/spring/RedisModulesConfiguration.java

```diff
@@ -109,30 +125,88 @@ public RediSearchIndexer redisearchIndexer(ApplicationContext ac) {
     return new RediSearchIndexer(ac);
   }
 
+  @Bean(name = "djlImageFactory")
+  public ImageFactory imageFactory() {
+    return ImageFactory.getInstance();
+  }
+
+  @Bean(name = "djlImageEmbeddingModelCriteria")
+  public Criteria<Image, byte[]> imageEmbeddingModelCriteria() {
+    return Criteria.builder()
+        .setTypes(Image.class, byte[].class) //
+        .optEngine("PyTorch")  //
+        .optModelUrls("djl://ai.djl.pytorch/resnet18_embedding") //
+        .build();
+  }
+
+  @Bean(name = "djlImageEmbeddingModel")
+  public ZooModel<Image, byte[]> imageModel(
+      @Qualifier("djlImageEmbeddingModelCriteria") Criteria<Image, byte[]> imageEmbeddingModelCriteria)
+      throws MalformedModelException, ModelNotFoundException, IOException {
+    return ModelZoo.loadModel(imageEmbeddingModelCriteria);
+  }
+
+  @Bean(name = "djlDefaultImagePipeline")
+  public Pipeline defaultImagePipeline() {
+    return new Pipeline() //
+        .add(new CenterCrop()) //
+        .add(new Resize(224, 224)) //
+        .add(new ToTensor());
+  }
+
+  @Bean(name = "djlSentenceTokenizer")
+  public HuggingFaceTokenizer sentenceTokenizer() {
+    Map<String, String> options = Map.of("maxLength", "768", "modelMaxLength", "768");
+    return HuggingFaceTokenizer.newInstance("sentence-transformers/all-mpnet-base-v2", options);
```

Comment:
Maybe we can use properties for these values: 
`hugging.face.tokenizer = sentence-transformers/all-mpnet-base-v2`